src/WikiService.Models.Dto/Requests/Rubric/Filters/GetRubricFilter.cs

```diff
@@ -0,0 +1,18 @@
+﻿using LT.DigitalOffice.Kernel.Requests;
+using Microsoft.AspNetCore.Mvc;
+using System;
+
+namespace LT.DigitalOffice.WikiService.Models.Dto.Requests.Rubric.Filters
+{
+  public record GetRubricFilter : BaseFindFilter
+  {
+    [FromQuery(Name = "rubricId")]
+    public Guid? RubricId { get; set; }
```

Comment:
get request can not have nullable the entity id

Reply:
Done